--- ./Makefile.pmk~	2005-01-10 01:34:37.000000000 +0200
+++ ./Makefile.pmk	2005-06-22 15:10:36.000000000 +0300
@@ -93,10 +93,11 @@
 	fi
 
 install::
-	$(INSTALL) -CvS -o $(OWNER) -g $(GROUP) -m 0555 $(LIB_STATIC) $(PREFIX)/lib
-	$(INSTALL) -CvS -o $(OWNER) -g $(GROUP) -m 0555 $(LIB_SHARED_V) $(PREFIX)/lib
-	$(INSTALL) -CvS -o $(OWNER) -g $(GROUP) -m 0444 memcache.h $(PREFIX)/include
-	cd $(PREFIX)/lib ; $(LN) -sf $(LIB_SHARED_V) $(LIB_SHARED)
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/{lib,include}
+	$(INSTALL) $(LIB_STATIC) $(DESTDIR)$(PREFIX)/lib
+	$(INSTALL) $(LIB_SHARED_V) $(DESTDIR)$(PREFIX)/lib
+	$(INSTALL) memcache.h $(DESTDIR)$(PREFIX)/include
+	cd $(DESTDIR)$(PREFIX)/lib; $(LN) -sf $(LIB_SHARED_V) $(LIB_SHARED)
 
 # A release engineering target
 release:	Makefile
